pytest-embedded-idf/tests/test_idf.py

```diff
@@ -682,6 +682,70 @@ def test_idf_hard_reset_and_expect(dut):
     result.assert_outcomes(passed=1)
 
 
+def test_select_to_run():
+    from pytest_embedded_idf.unity_tester import IdfUnityDutMixin
+
+    assert IdfUnityDutMixin._IdfUnityDutMixin__select_to_run(
```

Comment:
```suggestion
    assert IdfUnityDutMixin._select_to_run(
```

let's name it `_select_to_run`, and update the usage like this...

the `__` is somehow not recommended in python, since it refers to "magic functions"